web/themes/custom/novel/templates/components/full-event.html.twig

```diff
@@ -29,5 +29,14 @@
     } only %}
   </section>
 
+  {% if event_instances %}
+    <section class="event-list-stacked">
+      <h2 class="event-list-stacked__heading">{{ 'Upcoming events'|t }}</h2>
+      {% for instance in event_instances %}
+          {{ instance }}
+      {% endfor %}
+    </section>
+  {% endif %}
+
```

Comment:
Consider if the instances should be shown before the paragraphs.

I would have expected it to be below as I consider the paragraphs a part of the event "content" while the list of instances is something extra but I have not been part of the conversation with the business so you know best here.

Reply:
Yeah i understand. 

We agreed to put it above the paragraphs to begin with, but maybe they will change their mind when they see it 🤷